themes/bootstrap3/theme.config.php

```diff
@@ -61,5 +61,13 @@
             'layoutClass' => 'VuFind\View\Helper\Bootstrap3\LayoutClass',
             'search' => 'VuFind\View\Helper\Bootstrap3\Search'
         ]
+    ],
+    'icons' => [
```

Comment:
Another TODO item: once we're happy that we have all the functionality we need, we should add comments here (or somewhere) explaining the config format.

Reply:
This seems good to me. One suggestion, though: I don't think prefix needs to be a font-specific setting. It could be a prefix that gets applied to all icon names within the set. This could apply to filenames for files, or anchors for SVGs. It just defaults to empty. This way, you can use it to improve readability when everything has the same prefix, but you can also ignore it if you don't need it. You could potentially apply the prefix to the icon name inside the view helper so that you don't have to deal with it in the templates at all. Of course, I don't insist on this -- just a thought. What do you think?